_posts/2023-02-09-january-rubygems-updates.md

```diff
@@ -0,0 +1,78 @@
+- sped up resolution by properly merging incompatibility ranges - [#6215](https://github.com/rubygems/rubygems/pull/6215).
+- turned `--ext` option of bundle gem into a String. and deprecated usage without explicit value - [#6144](https://github.com/rubygems/rubygems/pull/6144).
+- enhanced `bundle open` command to allow opening subdir/file for a gem - [#6146](https://github.com/rubygems/rubygems/pull/6146).
+
+In addition to that, we made the following improvements and fixes(see the changelog for more information:
+
+- fixed flakiness on `Ruby 3.2` and Windows and speed up some specs - [#6321](https://github.com/rubygems/rubygems/pull/6321).
+- updated the list of `SPDX` licence identifiers - [#6310](https://github.com/rubygems/rubygems/pull/6310).
+- added tests for old lockfile with new ruby version - [#6317](https://github.com/rubygems/rubygems/pull/6317).
+- fixed packages for external platforms being introduced in lockfile when Bundler retries resolution - [#6285](https://github.com/rubygems/rubygems/pull/6285).
+
+In January, RubyGems gained [158 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2023-01-01%7D...master@%7B2023-01-31%7D), contributed by 14 authors. There were 3,450 additions and 2,535  deletions across 112 files.
+
+
+## RubyGems.org News
+
+In January, RubyGems.org saw several bug fixes and updates, some of which include the following:
+
+- added Webauthn Verification authenticate endpoint - [#3331](https://github.com/rubygems/rubygems.org/pull/3331).
+- moved browser tests to standard Rails system tests- [#3374](https://github.com/rubygems/rubygems.org/pull/3374).
+- added patch for mfa_expires_at edge condition - [#3357](https://github.com/rubygems/rubygems.org/pull/3357).
+- used simplified version of GitHub button JS. - [#3348](https://github.com/rubygems/rubygems.org/pull/3348).
+- introduced timeout on OTP page - [#3325](https://github.com/rubygems/rubygems.org/pull/3325).
+- added telemetry to capture MFA login durations - [#3376](https://github.com/rubygems/rubygems.org/pull/3376).
+
+This month, RubyGems.org gained [86 new commits](https://github.com/rubygems/rubygems.org/compare/master@%7B2023-01-01%7D...master@%7B2023-01-31%7D), contributed by 12 authors. There were 584 additions and 395 deletions across 46 files.
+
+As always, we continue to fix bugs, review and merge PRs and reply to support tickets.
+
+## Thank you
+
+Thank you to all the contributors of RubyGems and RubyGems.org for this month! Your contributions are greatly appreciated, and we are grateful for your support.
+
+### Contributors to RubyGems:
+
+- [@composerinteralia](https://github.com/composerinteralia) Daniel Colson
+- [@deivid-rodriguez](https://github.com/deivid-rodriguez) David Rodríguez
+- [@hsbt](https://github.com/hsbt) Hiroshi SHIBATA
+- [@yoka](https://github.com/yoka) Jesse Ikonen
+- [@simi](https://github.com/simi?tab=overview&from=2023-02-01&to=2023-02-09) Josef Šimánek
```

Comment:
Any reason to have those special inks instead of just link to profile?

Reply:
Oh no, that's a typo, I will fix it now.